base/applications/iexplore/iexplore.inf

```diff
@@ -16,7 +16,10 @@ HKCR,"CLSID\%CLSID_InternetExplorer%\LocalServer32",,,"""%16422%\Internet Explor
 HKCR,"CLSID\%CLSID_InternetExplorer%\ProgID",,,"InternetExplorer.Application.1"
 HKCR,"CLSID\%CLSID_InternetExplorer%\VersionIndependentProgID",,,"InternetExplorer.Application"
 
-HKCR,"CLSID\%CLSID_Internet%\DefaultIcon",,,"shdoclc.dll,-190"
+HKCR,"CLSID\%CLSID_Internet%",,,"Internet"
```

Comment:
I would comment these out as long as it is not working, that saves us from receiving 30 tickets that it does not work.

Reply:
Commented out. But `shell32` keeps the icon shown.

![VirtualBox_ReactOS_18_07_2023_17_17_45](https://github.com/reactos/reactos/assets/2107452/0119a90c-46cf-46c8-8d91-09927ba280d9)

Reply:
Double-clicking action is fixed.